### PR TITLE
feat: add nochange option which will get the current release if no ch…

### PR DESCRIPTION
**Pull request by joshughes**
Allow us to rerun builds or have multiple builds in parallel that compile different artifacts but share a release number

### CODE REVIEW

File: cmd/semantic-release/main.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ func main() {
 	slug := flag.String("slug", os.Getenv("TRAVIS_REPO_SLUG"), "slug of the repository")
 	ghr := flag.Bool("ghr", false, "create a .ghr file with the parameters for ghr")
 	noci := flag.Bool("noci", false, "run semantic-release locally")
+	nochange := flag.Bool("nochange", false, "allow semver to run where the current commit is already a tagged release")
```
Comment (jbcpollak) [inline cmd/semantic-release/main.go]: I would clarify what 'allow' means - something like 'don't return an error code when the calculated version has already been tagged' might be a bit clearer

**Comment by jbcpollak** [comment]
@joshughes looks good to me, I just find the help message a bit counter intuitive.

**Comment by joshughes** [comment]
@jbcpollak I struggled a bit with what to put in the help message... I will try to think of a better way to phrase is... 

Do you have any ideas?

File: cmd/semantic-release/main.go (hunk 1 above)
Comment (joshughes) [reply cmd/semantic-release/main.go]: Thanks I struggled a bit here on how to phrase this

**Pull request merged**